sqlitecluster/SQLiteNode.cpp

```diff
@@ -2275,7 +2271,7 @@ bool SQLiteNode::peekPeerCommand(SQLiteNode* node, SQLite& db, SQLiteCommand& co
             _queueSynchronizeStateless(command.request.nameValueMap,
                                        command.request["name"],
                                        command.request["peerName"],
-                                       SToInt(command.request["state"]),
+                                       node->_state,
```

Comment:
See my comment above, not sure that state is even needed for this function call, but isn't it now possible that the peer->state changes before this command is peeked? the old implementation was using the info from the command which could not change.

Reply:
This is probably *more* accurate now, I think, because the old one could potentially be out-of-date. It's not super critical either way, as it's only for logging.

Reply:
cool